app/traffic_cloud_utils/plotting/visualization.py

```diff
@@ -29,15 +29,12 @@
 
 
 
-def road_user_traj(fig, filename, fps, homographyFile, roadImageFile):
+def road_user_traj(filename, homographyFile, roadImageFile, save_path, objs_to_plot=None, plot_cars=False):
     """
     Plots all road-user trajectories.
     """
-
     homography = inv(loadtxt(homographyFile))
```

Comment:
AFAIK, homography projects from pixels into the world space. We're using this to project back from world space into pixel space, so we need the inverse matrix.